modules/ROOT/pages/functions/faq.adoc

```diff
@@ -0,0 +1,258 @@
+Serverless Functions FAQ
+========================
+
+Serverless Functions frequently asked questions.
+
+
+== What are Serverless Functions?
+
+Serverless Functions are a collection of tools that enable developers to create
+and run functions as a Knative Service on Kubernetes. The major components of
+this project are listed here.
+
+* Function runtimes for Go, Node.js and Quarkus
+* Buildpacks for Go, Node.js and Quarkus
+* A CLI for creating and managing functions as a Knative Service
+* Knative Serving and Eventing as the platform on which it all runs
+
+== How mature is the Serverless Functions Project?
+
+OpenShift Serverless Functions are currently in Developer Preview.
+
+== How does this project relate to Knative?
+
+Knative is the target deployment platform for Serverless Functions.
+
+== How do I Use Serverless Functions?
+
+For Function projects, a developer uses the `kn-func` CLI for project creation
+and deployment.  Developers use this CLI to create new function projects and
+deploy them to a Kubernetes cluster running Knative Serving and Eventing.
+During deployment, the developer's function code is combined with a runtime
+framework (for example Quarkus or Node.js) using the CNCF Buildpack APIs to
+create a runnable OCI container.
+
+To get started using Serverless Functions now, please follow the [step by step
+tutorial](tutorial.md).
```

Comment:
All of this just seems like duplication of content that should is already or should be included in its own module; I really don't like the idea of jamming this all into an FAQ tbh